setup.py

```diff
@@ -76,6 +76,10 @@ def main():
         'tensorflow == 1.15.4'
     ]
 
+    horovod_pypi_dep = [
+        'horovod[tensorflow] == 0.21.3'
```

Comment:
(I have not checked the pypi repo) how does that works when `tensorflow-gpu` is needed, is this `horovod[tensorflow]` still working as expected ?

Reply:
Maybe you want to take a look here https://horovod.readthedocs.io/en/stable/install.html
There is only `horovod[tensorflow]` for TensorFlow bindings and should be the same as `HOROVOD_WITH_TENSORFLOW=1 pip install horovod`.